WalletWasabi/Mono/StringCoda.cs

```diff
@@ -186,38 +186,36 @@ private static IEnumerable<string> CreateWrappedLinesIterator(string self, IEnum
 				yield return string.Empty;
 				yield break;
 			}
-			using (IEnumerator<int> eWidths = widths.GetEnumerator())
+			using IEnumerator<int> eWidths = widths.GetEnumerator();
```

Comment:
wtf? `IEnumerator` is not an `IDisposable`

Reply:
Ok, I was wrong, an `IEnumerator` is `IDisposable`.

Reply:
Yes, it's not `IEnumerable`